packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2177,6 +2177,20 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             ? this.summarizer.summarizeOnDemand(...args)
             : this.summaryManager.summarizeOnDemand(...args);
```

Comment:
We should add a case where this.summaryManager isn't defined, similar to `enqueueSummarize`

Reply:
It's already there.. just not showing in this diff.